builder/js/ncf.js

```diff
@@ -1330,6 +1337,7 @@ $scope.onImportFileChange = function (fileEl) {
         // If we were cloning a technique, remove its 'clone' state
         savedTechnique.isClone    = false;
         $scope.originalTechnique  = angular.copy(savedTechnique);
+        $scope.originalTechnique.resources = undefined;
```

Comment:
can you add a comment here ? to explain the resources will be reset by resource manager if original technique resources are undefined ?